resources/docker-compose.yml

```diff
@@ -11,18 +11,18 @@ services:
       - 50000:50000
     container_name: jenkins
     environment:
-      - JENKINS_JAVA_OPTS=-Xmx60g -Xss4m
+      - JENKINS_JAVA_OPTS=-Xms33g -Xmx65g -Dhudson.model.ParametersAction.keepUndefinedParameters=true -XX:+UseG1GC -XX:+ExplicitGCInvokesConcurrent -XX:+ParallelRefProcEnabled -XX:+UseStringDeduplication -XX:+UnlockExperimentalVMOptions -XX:G1NewSizePercent=20 -XX:+UnlockDiagnosticVMOptions -XX:G1SummarizeRSetStatsPeriod=1
       - CASC_RELOAD_TOKEN=reloadPasswordHere
     volumes:
       - /var/lib/jenkins:/var/jenkins_home
     deploy:
       resources:
         limits:
-          cpus: '32'
-          memory: '64g'
+          cpus: '34'
+          memory: '67g'
         reservations:
-          cpus: '32'
-          memory: '64g'
+          cpus: '34'
+          memory: '67g'
```

Comment:
Thanks @prudhvigodithi .

Can we keep the cpu and xms to 32/32g?
And xmx 64g?

So the host has a few more resources even if docker is throttling the instance. Thanks.

Reply:
Updated the PR to have some bandwidth for the server process. Thanks